autoload/ale/python.vim

```diff
@@ -25,8 +25,7 @@ function! ale#python#FindProjectRootIni(buffer) abort
         \|| filereadable(l:path . '/tox.ini')
         \|| filereadable(l:path . '/mypy.ini')
         \|| filereadable(l:path . '/pycodestyle.cfg')
-        \|| filereadable(l:path . '/flake8.cfg')
```

Comment:
`python-language-server` discovers paths by looking for `flake8.cfg`, so I'd keep this. It's mentioned here: https://pypi.org/project/python-language-server/

Reply:
It appears to be a typo in the README of `python-language-server`.

https://github.com/palantir/python-language-server/blob/96e08d85635382d17024c352306c4759f124195d/pyls/config/flake8_conf.py#L10

The commit which introduced the original introduced the change for reading `flake8` but documented it incorrectly (https://github.com/palantir/python-language-server/commit/3c1f18361179f3a687bdef6209e136a10bbf043f).

The `python-language-server` has limited contributions to the project to those who have contributed in the past — I am unable to open an issue nor a pull request to address this discrepancy 😞 .